File: java/java-arithmetic/src/main/java/com/leetcode/first/J70.java
package com.leetcode.first;

public class J70 {

    public static void main(String[] args) {
        System.out.println(new J70().climbStairs(3));

    }

    public int climbStairs(int n) {
        int first =0, second =1;
        int result =0;

        for (int i=0;i<=n;i++){
            first = second;
            second = result;
            result = first + second;
        }

        return result;

    }
}

File: test-python/leetcode/J70.py
class Solution:
    def climbStaeirs(self, n: int) -> int:
        first =0
        second = 1
        result = 0
        for i in range(n+1):
            print(i)
            first = second
            second = result
            result = first +second
        
        return result


print(Solution().climbStaeirs(3))